modules/integration/tests-integration/tests-backend/src/test/resources/oas/v2/apiData.json

```diff
@@ -48,8 +48,8 @@
     "accessControlAllowMethods": ["GET", "PUT", "POST", "DELETE", "PATCH", "OPTIONS"]
   },
   "workflowStatus": null,
-  "createdTime": "2019-09-17 17:49:55.251",
```

Comment:
shall we remove createdTime and lastUpdatedTime. since this was not updated from rest API level.